envoy-control-runner/src/main/resources/application-docker.yaml

```diff
@@ -2,4 +2,8 @@ envoy-control:
   source:
     consul:
       host: consul
-      port: 8500
\ No newline at end of file
+      port: 8500
+
+chaos:
```

Comment:
I have a feeling you will soon have to migrate these properties to a deeper properties tree, but it's an incubating feature so it's fine.